hnn_core/gui/gui.py

```diff
@@ -943,6 +946,54 @@ def _get_evoked_widget(name, layout, style, location, data=None,
     return drive, drive_box
 
 
+def _get_tonic_widget(name, layout, style, data=None):
+    default_data = {
+        'amplitude': 0,
+        't0': 0,
+        'tstop': 0,
+        'seedcore': 14,
+    }
+
+    cell_types = ['L5_pyramidal', 'L2_pyramidal', 'L5_basket', 'L2_basket']
+
+    if isinstance(data, dict):
+        default_data.update(data)
+    kwargs = dict(layout=layout, style=style)
+
+    amplitudes, start_times, stop_times = dict(), dict(), dict()
+    for cell_type in cell_types:
+        if cell_type in data:
+            default_data.update(data[cell_type])
+        amplitudes[cell_type] = BoundedFloatText(
+            value=copy.deepcopy(default_data['amplitude']),
+            description=cell_type, min=0, max=1e6, step=0.01, **kwargs)
+        start_times[cell_type] = BoundedFloatText(
+            value=copy.deepcopy(default_data['t0']), description=cell_type,
+            min=0, max=1e6, step=0.01, **kwargs)
+        stop_times[cell_type] = BoundedFloatText(
+            value=copy.deepcopy(default_data['tstop']), description=cell_type,
+            min=-1, max=1e6, step=0.01, **kwargs)
```

Comment:
Is there a reason why we set independent start/stop times for each cell type?

Reply:
you mean the range? or the data values?

Reply:
Like, I can currently add a single tonic bias that starts and stops at different times depending on the cell type. If someone wants to set different dynamics for different cell types, they should just create separate baises that correspond one-to-one with the different cell types.

Reply:
agree about that ... the GUI should reflect what the `hnn-core` API says, which is one cell-type per bias

Reply:
Wait, a bias is only supposed to target a single cell type?